src/modules/assessments/components/AssessmentForm.tsx

```diff
@@ -143,16 +150,16 @@ const AssessmentForm: React.FC<Props> = ({
   );
 
   // Local values that may be referenced by the FormDefinition
-  const localConstants = useMemo(
+  const localConstants: AssessmentLocalConstants = useMemo(
     () => ({
-      entryDate: enrollment.entryDate,
-      exitDate: enrollment.exitDate,
-      projectName: enrollment.project.projectName,
-      clientFirstName: client.firstName,
-      clientMiddleInitial: client.middleName ? client.middleName[0] : '',
-      clientLastName: client.lastName,
```

Comment:
These were added for https://github.com/open-path/Green-River/issues/6045 but never used for the intended purpose (or at all, at least in prod).